libs/domains/environments/feature/src/lib/hooks/use-delete-environment/use-delete-environment.ts

```diff
@@ -17,6 +19,12 @@ export function useDeleteEnvironment({ projectId }: { projectId: string }) {
     meta: {
       notifyOnSuccess: {
         title: 'Your environment is being deleted',
+        ...(logsLink
```

Comment:
I wonder if this label should be not required for all actions, this should improve our UX to facilitate the navigation to the logs